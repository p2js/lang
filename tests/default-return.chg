fn foo() {
    let x = 1;
}
fn main() {
    foo();
}
