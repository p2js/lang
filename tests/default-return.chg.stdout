fallback @ 2:13 2:14
expr @ 2:13 2:14 => i32
expr @ 1:10 4:0 => ()
expr @ 5:5 5:8 => fn foo(...)
expr @ 5:5 5:10 => ()
expr @ 4:11 7:1 => ()



define void @foo() {
prologue:
%l.0 = alloca i32
br label %bb.0

bb.0:
store i32 1, i32* %l.0
ret void

}

define void @main() {
prologue:
br label %bb.0

bb.0:
%l.0 = call {} bitcast(void()* @foo to {}()*)()
br label %bb.1

bb.1:
ret void

}


